layouts/ensign-pricing/single.html

```diff
@@ -11,7 +11,7 @@
 {{ define "main" }}
 <div class="max-w-7xl mx-auto my-8 px-4">
   <div class="mt-14">
-    {{ .Content | .RenderString }}
```

Comment:
`.RenderString` is no longer needed to render markup to HTML. Also, it caused some HTML to be displayed inside of `<pre>` tags.